### PR TITLE
Fix attach, change webhook name, add rio pods and rio kill

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: cli/cmd/run/run.go [hunk 1]
```diff
@@ -28,9 +30,12 @@ func (r *Run) Run(ctx *clicontext.CLIContext) error {
 		isatty.IsTerminal(os.Stderr.Fd()) &&
 		isatty.IsTerminal(os.Stdin.Fd())
 
+	app, version := services.AppAndVersion(service)
+	svcName := fmt.Sprintf("%s@%s", app, version)
```
Comment (cbron) [inline cli/cmd/run/run.go]: I'm confused after so many changes, are we guaranteeing version now ? Would this never end up as `app@`

File: cli/main.go [hunk 2]
```diff
@@ -196,6 +201,10 @@ func main() {
 			"Apply a rio file",
 			appName+" up [OPTIONS]",
 			""),
+		builder.Command(&kill.Kill{},
+			"Kill pods or pods from services",
```
Comment (cbron) [inline cli/main.go]:
```suggestion
			"Kill pods or services",
```
unless you really wanted to specify that

File: cli/cmd/run/run.go (hunk 1 above)
Comment (StrongMonkey) [reply cli/cmd/run/run.go]: Yeah this method will always return app and version. The bug in attach is that the parse logic doesn't take the real service name. it only takes app@version.

**Comment by StrongMonkey** [comment]
addressing https://github.com/rancher/rio/issues/691.

**Pull request merged**